src/pages/about-us/assets/about-us.module.scss

```diff
@@ -0,0 +1,144 @@
+@import '../../../../src/scss/breakpoints.module';
+@import '../../../../src/scss/fonts.module';
+
+body {
```

Comment:
I am not sure if applying styles to the body in that file is okay

Reply:
true, i moved it to App.tsx so it's applied globally